springwolf-examples/springwolf-amqp-example/src/test/resources/asyncapi.json

```diff
@@ -119,6 +140,69 @@
           "bindingVersion": "0.3.0"
         }
       }
+    },
+    "queue-delete": {
+      "address": "queue-delete",
+      "messages": {
+        "io.github.springwolf.examples.amqp.dtos.GenericPayloadDto": {
+          "$ref": "#/components/messages/io.github.springwolf.examples.amqp.dtos.GenericPayloadDto"
+        }
+      },
+      "bindings": {
+        "amqp": {
+          "is": "queue",
+          "queue": {
+            "name": "queue-delete",
+            "durable": true,
+            "exclusive": false,
+            "autoDelete": false,
+            "vhost": "/"
+          },
+          "bindingVersion": "0.3.0"
+        }
+      }
+    },
+    "queue-read_#_CRUD-topic-exchange-2": {
```

Comment:
Does this make sense?

This is an entry of type `is=routingKey`, which belongs to an exchange.
However the routingKey is not specified.
Instead, the routingKey is part of the channelIdentifier - which has no special meaning.

Does the spec require the field routingKey?
Similarly, is the target queue (binding) needed as well, especially since the routingKey can also be a pattern.